_posts/2016-xx-xx-release-0.16.md

```diff
@@ -119,6 +119,116 @@ If you are using the low-level qcprot algorithm your self intead of our provided
 wrappers you have to change your code since the API has changed. For more see
 the [CHANGELOG].
 
+## MemoryReader: Reading trajectories from memory
+
+MDAnalysis typically reads trajectories from files on-demand, so that it can efficiently deal with large trajectories - even those that do not fit in memory. However, in some cases, both for convenience and for efficiency, it can be an advantage to work with trajectories directly in memory. In this release, we have introduced a MemoryReader, which makes this possible.
+
+The MemoryReader works with numpy arrays, using the same format as that used by for instance `DCDReader.timeseries()`. You can create a Universe directly from such an array:
+
+```python
+from MDAnalysis import Universe
+from MDAnalysisTests.datafiles import DCD, PDB_small
+from MDAnalysis.coordinates.memory import MemoryReader
+
+# Create a Universe using a DCD reader
+universe = Universe(PDB_small, DCD)
+
+# Extract coordinates
+coordinates = universe.trajectory.timeseries()
+
+# Create a new Universe directly from these coordinates
+# using the MemoryReader
+universe2 = Universe(PDB_small, coordinates,
+                     format=MemoryReader)
+```
+
+The MemoryReader will work just as any other reader. In particular, you can iterate over it as usual, or use the `.timeseries()` method to retrieve a reference to the raw array in any format:
+
+```python
+coordinates_fac = universe2.trajectory.timeseries(format='fac')
+```
+
+Certain operations can be speeded up by moving a trajectory to memory. To facilitate this operation, the constructor of `Universe` takes an `in_memory` flag which will automatically convert any trajectory to a MemoryReader:
+
+```python
+universe = Universe(PDB_small, DCD, in_memory=True)
+```
+
+Likewise, the `rms_fit_trj` function in the analysis/align.py module also has an `in_memory` flag, allowing it to do in-place alignments in memory.
+
+
+## Incorporation of the ENCORE ensemble similarity library
+
+The ENCORE ensemble similarity library has been integrated with MDAnalysis. It implements a variety of techniques for calculating similarities between structural ensembles (trajectories), as described in this publication:
```

Comment:
Mention the module and link to the docs:
```markdown
The **ENCORE** ensemble similarity library has been integrated with MDAnalysis as
[MDAnalysis.analysis.encore](http://www.mdanalysis.org/mdanalysis/documentation_pages/analysis/encore.html).
```
-----

Note: the link to the docs is broken because of MDAnalysis/mdanalysis#1047